charts/pocket/pocket-servicer-overrides.yaml

```diff
@@ -0,0 +1,10 @@
+config:
```

Comment:
Why did we need to add an override for this?

Reply:
@okdas Could you chime in on this approach? It looks good to me but I'd like a second set of 👀

Reply:
@Olshansk Using a separate file to override values is just one of the patterns for utilizing Helm. This file was originally introduced in #613. It's not necessary to have this file, and we actually set the values [without relying on the override file](https://github.com/pokt-network/pocket/blob/main/build/localnet/Tiltfile#L216) (though it's still included) on LocalNet. I believe it's beneficial to have this file as a reference for configuring the service with Helm. But this is not something we utilize on DevNets, for example.

Reply:
@adshmh Could you add a comment at the top of this file that reflects yours and @okdas's comment? Specifically including things like:
```
- This is an override of the shared configuration: <link>
- This is a reference of how we can utilize Helm
- COnfigs can also be overrident without an explicit config file: <link>
```

Reply:
Thank you for the review. Updated the PR.

Reply:
Thank you for the review. Updated the PR.